lib/src/model.dart

```diff
@@ -4054,6 +4056,109 @@ abstract class ModelElement extends Canonicalization
     }
   }
 
+  /// Replace &#123;@youtube ...&#125; in API comments with some HTML to embed
+  /// a YouTube video.
+  ///
+  /// Syntax:
+  ///
+  ///     &#123;@youtube WIDTH HEIGHT URL&#125;
+  ///
+  /// Example:
+  ///
+  ///     &#123;@youtube 560 315 https://www.youtube.com/watch?v=oHg5SJYRHA0&#125;
+  ///
+  /// Which will embed a YouTube player into the page that plays the specified
+  /// videos.
+  ///
+  /// The width and height must be integers specifying the dimensions of the
+  /// video in pixels.
+  ///
+  /// The following YouTube URL formats are supported:
+  ///
+  ///  * https://www.youtube.com/watch?v=oHg5SJYRHA0 (as shown in the browser's
+  ///    address bar)
+  ///  * https://youtu.be/oHg5SJYRHA0 (as shown in the "Share" dialog)
+  ///  * https://www.youtube.com/embed/oHg5SJYRHA0 (as shown in the "Share ->
+  ///    Embed" dialog)
```

Comment:
we should standardise on one to keep things consistent

Reply:
Sure, you want to offer an opinion on what the one URL format should be that we support? I'd vote for either https://www.youtube.com/watch?v=oHg5SJYRHA0 or https://youtu.be/oHg5SJYRHA0. The former is easier accessible for people wanting to embed something (just copy the URL from the address bar), the second one looks cleaner.

Reply:
Settled on https://www.youtube.com/watch?v=oHg5SJYRHA0.